.ibm/pipelines/utils.sh

```diff
@@ -712,3 +726,45 @@ oc_login() {
   export K8S_CLUSTER_ROUTER_BASE=$(oc get route console -n openshift-console -o=jsonpath='{.spec.host}' | sed 's/^[^.]*\.//')
 }
 
+# Functions needed to deploy operator instances
```

Comment:
Since those functions are specific only for operator, I think you can move it to jobs/operator.sh